app/models/account_relationship_severance_event.rb

```diff
@@ -26,9 +31,14 @@ class AccountRelationshipSeveranceEvent < ApplicationRecord
 
   before_create :set_relationships_count!
 
+  def relationships_count
```

Comment:
Is this accessed from anywhere after these changes?

Reply:
Indeed not.